apps/sms/views/shared/js/startup.js

```diff
@@ -108,16 +98,33 @@ var Startup = {
 
       MessageManager.init();
       Navigation.init();
+      InboxView.init();
+
+      InboxView.once('fully-loaded', () => {
+        window.performance.mark('fullyLoaded');
+
+        App.setReady();
+      });
+
+      InboxView.once('visually-loaded', () => {
```

Comment:
We can probably call ```_lazyLoadInit``` here and remove one more listener below, for this we'll have to change ```_lazyLoadInit``` a bit so that it can check if lazy loading already started.

I was thinking that having two listeners should not affect performance much